src/test/resources/testbib/complex.bib

```diff
@@ -296,8 +296,6 @@ @Comment{jabref-meta:
 1 KeywordGroup:DynamicGroup\;0\;author\;Churchill\;0\;0\;;
 }
 
-@Comment{jabref-meta: groupsversion:3;}
```

Comment:
We have a test `integrationTestGroupTree` https://github.com/JabRef/jabref/pull/2112/files#diff-837597ff5fafbe4e71d5733d490e3454R1479 which still have the ̀`groupsversion:3` string. But since we no longer write the the `groupsversion` information, the roundtrip tests for complex.bib fail. This is why I removed it.